.github/workflows/pr-preview.yml

```
@@ -0,0 +1,103 @@
+    if: github.event.action == 'closed'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout gh-pages branch
+        uses: actions/checkout@v4
+        with:
+          ref: gh-pages
+
+      - name: Remove preview directory
+        run: |
+          rm -rf pr-preview/${{ github.event.pull_request.number }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)